examples/private_connection/README.md

```diff
@@ -0,0 +1,61 @@
+# Private Connectivity Example
+
+This example shows how to configure a forseti to use only private connection between server, client and GCP services (e.g. Storage or Cloud SQL).
+
+This example deploys the following:
+
+1. Forseti infrastructure
+   * CloudSQL Database
+   * Forseti Server GCS Bucket
+   * Forseti Client GCS Bucket
+   * Forseti Client VM
+   * Forseti Server IAM Service Account
+   * Forseti Client IAM Service Account
+2. Forseti on GKE - forseti-on-gke
+3. DNS and NAT configuration for private communication between forseti server and GCP services
+
+## Requirements
+
+Before this example can be used, you have to ensure that the following pre-requisites are fulfilled:
+
+* Terraform is [installed](#software-dependencies) on the machine where the script is executed.
+* GCP Credentials should be [setup](#setup-credentials)
+* (OPTIONAL) Service APIs should be [active](#enable-apis) on the project that you plan to deploy the example
+
+The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active.
+
+### Software Dependencies
+
+#### Terraform and Plugins
+
+* [Terraform](https://www.terraform.io/downloads.html) 0.12+
+* [Terraform Provider for GCP](https://www.terraform.io/docs/providers/google/index.html) v2.9+
+
+### Setup Credentials
+
+When [configuring Terraform credentials for GCP](https://www.terraform.io/docs/providers/google/guides/getting_started.html#adding-credentials) you should take care that the identity has proper roles on the example's project. In addition to the [roles](https://github.com/forseti-security/terraform-google-forseti#iam-roles) required for the core module function, the identity must have these roles for this example.
+
+* roles/iam.serviceAccountAdmin
+* roles/iam.serviceAccountKeyAdmin
+* roles/compute.networkAdmin
+* roles/resourcemanager.projectIamAdmin (only required if `service_account` is set to `create`)
+
+### Enable APIs
+
+In order to operate the following APIs should be active on the example's project:
+
+* Compute Engine API - compute.googleapis.com
+* Cloud Resource Manager API - cloudresourcemanager.googleapis.com
+* Service Usage API - serviceusage.googleapis.com
+* Service Networking API - servicenetworking.googleapis.com
+* Cloud DNS API - dns.googleapis.com
+
+## Inputs
+
+You can setup the inputs by renaming and editing the `terraform.tfvars2` file to `terraform.tfvars` or use alternative methods.
+
+| Name | Description | Type | Default | Required |
```

Comment:
Run `make generate_docs` to auto-populated the input and output sections. You need the
`<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->` and `<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->` markers as in other README.md files.

Reply:
done